prombench/docs/kind.md

```diff
@@ -0,0 +1,87 @@
+# Prombench in KIND
+
+Run prombench tests in [Kubernetes In Docker](https://kind.sigs.k8s.io/).
+
+## Setup prombench
+
+1. [Create the KIND cluster](#create-the-kind-cluster)
+2. [Deploy monitoring components](#deploy-monitoring-components)
+
+### Create the Cluster Cluster
+
+- Create multi node KIND cluster
+- Set the following environment variables and deploy the cluster.
+
+```
+export CLUSTER_NAME=prombench
+export PR_NUMBER=<PR to benchmark against the selected $RELEASE>
+
+../infra/infra kind cluster create -v PR_NUMBER:PR_NUMBER -v CLUSTER_NAME:$CLUSTER_NAME \
+    -f manifests/cluster_kind.yaml
+```
+
+
+### Deploy monitoring components
+
+
+> Collecting, monitoring and displaying the test results and logs
+---
+
+- [Optional] If used with the Github integration generate a GitHub auth token.
+  - Login with the [Prombot account](https://github.com/prombot) and generate a [new auth token](https://github.com/settings/tokens).
+  - With permissions: `public_repo`, `read:org`, `write:discussion`.
+
+```
+export GRAFANA_ADMIN_PASSWORD=password
+export DOMAIN_NAME=prombench.prometheus.io // Can be set to any other custom domain or an empty string when not used with the Github integration.
+export OAUTH_TOKEN=<generated token from github or set to an empty string " ">
+export WH_SECRET=<github webhook secret>
+export GITHUB_ORG=prometheus
+export GITHUB_REPO=prometheus
+```
+
+- Deploy the [nginx-ingress-controller](https://github.com/kubernetes/ingress-nginx), Prometheus-Meta, Loki, Grafana, Alertmanager & Github Notifier.
+
+```
+../infra/infra kind resource apply -v CLUSTER_NAME:$CLUSTER_NAME -v DOMAIN_NAME:$DOMAIN_NAME \
+    -v GRAFANA_ADMIN_PASSWORD:$GRAFANA_ADMIN_PASSWORD \
+    -v OAUTH_TOKEN="$(printf $OAUTH_TOKEN | base64 -w 0)" \
+    -v WH_SECRET="$(printf $WH_SECRET | base64 -w 0)" \
```

Comment:
Hi, can you remind me what happens when we don't provide the `GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL` deployment var here? https://github.com/prometheus/test-infra/pull/391

Reply:
We can provide the GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL. Just to be generic we need SERVICEACCOUNT_CLIENT_EMAIL

Reply:
It breaks when we don't provide the GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL deployment var, right?